modules/siddhi-core/src/main/java/io/siddhi/core/table/record/AbstractRecordTable.java

```diff
@@ -68,8 +68,12 @@ public void init(TableDefinition tableDefinition, StreamEventFactory storeEventP
         this.recordTableHandler = recordTableHandler;
         this.storeEventPool = storeEventPool;
         init(tableDefinition, configReader);
+        init(tableDefinition, siddhiAppContext, storeEventCloner, configReader);
```

Comment:
Shall we rename the 2nd init? This is because both init are not doing the same thing.

Reply:
any suggestion for the name